Simplenote/Classes/SPTagsListViewController.m

```diff
@@ -483,48 +530,41 @@ - (void)removeTagAtIndexPath:(NSIndexPath *)indexPath {
         [appDelegate.noteListViewController update];
     }
     
-    BOOL lastTag = [self numTags] == 1;
+    BOOL lastTag = [self numberOfTags] == 1;
 
     if ([[SPObjectManager sharedManager] removeTag:tag] && !lastTag) {
-        
         [self.tableView beginUpdates];
-            [self.tableView deleteRowsAtIndexPaths:@[indexPath]
-                                  withRowAnimation:UITableViewRowAnimationLeft];
+        [self.tableView deleteRowsAtIndexPaths:@[indexPath] withRowAnimation:UITableViewRowAnimationLeft];
         [self.tableView endUpdates];
     } else {
         [self.tableView reloadData];
     }
 }
 
 - (void)renameTagAction:(Tag *)tag {
-    
     if (_renameTag) {
-        
-        SPTagListViewCell *cell = (SPTagListViewCell *)[self.tableView cellForRowAtIndexPath:[NSIndexPath indexPathForItem:[self rowForTag:_renameTag]
-                                                                                                                 inSection:kSectionTags]];
+        SPTagListViewCell *cell = [self cellForTag:_renameTag];
         [cell.tagNameTextField endEditing:YES];
     }
     
     _renameTag = tag;
```

Comment:
Accessing the ivar directly seems a little weird outside of its setter.  Maybe consider refactoring this into a setter method or using `self.`?